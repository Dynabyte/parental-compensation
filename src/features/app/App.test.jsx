import AppComponent from "./App.component";

describe("Test ParentalCompensationApp Component", () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ParentalCompensationApp />);
  });

  it("should render ParentalCompensationApp correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
