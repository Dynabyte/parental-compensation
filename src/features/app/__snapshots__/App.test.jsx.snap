// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test ParentalCompensationApp Component should render ParentalCompensationApp correctly 1`] = `
<styled.div>
  <GlobalStyleComponent />
  <styled.h3
    mt={30}
  >
    Vad behöver jag tänka på?
  </styled.h3>
  <Connect(Criteria) />
  <styled.h3>
    Prisbasbeloppet
  </styled.h3>
  <Connect(PriceBase) />
  <Connect(BasicSalaryComponent) />
  <Connect(CardComponent) />
</styled.div>
`;
